docs/USING_ACTIONS.md

```diff
@@ -0,0 +1,65 @@
+## Actions
+
+Each of the reducers: grid, pager, selection, etc have supporting actions. You may wish to dispatch these actions from other areas of your application outside of the grid. For instance, you could unselect all the rows of the grid that has checkbox column.
```

Comment:
Maybe "For instance, you could select all/deselect all rows in a grid from another component in your application"? is a better use case?